addons/web_editor/static/src/js/editor/odoo-editor/src/utils/serialize.js

```diff
@@ -20,7 +20,10 @@ export function serializeNode(node, nodesToStripFromChildren = new Set()) {
         let child = node.firstChild;
         while (child) {
             if (!nodesToStripFromChildren.has(child.oid)) {
-                result.children.push(serializeNode(child, nodesToStripFromChildren));
+                const serializedChild = serializeNode(child, nodesToStripFromChildren);
+                if (serializedChild) {
```

Comment:
What difference does this make, since `serializeNode` will throw an error if its return value is `undefined`?